scss/bitstyles/organisms/sidebar/_sidebar.scss

```diff
@@ -6,12 +6,13 @@
   width: settings.$large-width;
 }
 
+/* stylelint-disable declaration-empty-line-before */
```

Comment:
Given the size of the block here, maybe we're better of with "next-line" ignoring or inline ignoring. See https://stylelint.io/user-guide/ignore-code/

Just a thought.